src/stylesheets/components/_components.header.scss

```diff
@@ -44,24 +44,27 @@
 }
 
 .c-header__logo {
-  cursor: pointer;
+  cursor:pointer !important;
```

Comment:
!important isn't really good practice unless you really have to. Is it the case here?

Reply:
Seems not to work without it. I'll go through it again and see if there's somewhere else i need to change instead.